### PR TITLE
fixed creating multiple duplicate data objects

### PR DESCRIPTION
**Pull request by Makreig**
As the save button wasn't being locked out, if it was clicked multiple times then multiple duplicate objects would be created.

### CODE REVIEW

File: client/src/legacy/LeftAndMain.js [hunk 1]
```diff
@@ -443,8 +443,9 @@ $.entwine('ss', function($) {
 
       // set button to "submitting" state
       $(button).addClass('btn--loading loading');
+      $(button).attr('disabled', 'disabled');
```
Comment (ScopeyNZ) [inline client/src/legacy/LeftAndMain.js]:
```suggestion
      $(button).prop('disabled', true);
```

File: client/src/legacy/LeftAndMain.js [hunk 2]
```diff
@@ -462,6 +463,7 @@ $.entwine('ss', function($) {
 
       var clearButton = function() {
         $(button).removeClass('btn--loading loading');
+        $(button).attr('disabled', false);
```
Comment (ScopeyNZ) [inline client/src/legacy/LeftAndMain.js]:
```suggestion
        $(button).prop('disabled', false);
```

File: client/src/legacy/LeftAndMain.js [hunk 3]
```diff
@@ -443,8 +443,9 @@ $.entwine('ss', function($) {
 
       // set button to "submitting" state
       $(button).addClass('btn--loading loading');
+      $(button).attr('disabled', 'disabled');
 
-      if($(button).is('button')) {
+      if($(button).is('button')) {
```
Comment (ScopeyNZ) [inline client/src/legacy/LeftAndMain.js]:
```suggestion
      if($(button).is('button')) {
```

**Comment by Makreig** [comment]
> If there's a validation error, will the button be returned to its original state?

from what i can see, the buttons state is cleared whether there is an error or not, so it should be.

**Comment by Makreig** [comment]
I have no idea why this test has failed. anyone else know what's wrong?

**Comment by ScopeyNZ** [comment]
> I have no idea why this test has failed. anyone else know what's wrong?

There's an automated test that asserts the bundle you've included has been compiled correctly - but that's failing. You'll need to `yarn build` again from your branch and commit the new bundle.

**Pull request merged**